Sources/Jobs/JobQueueHandler.swift

```diff
@@ -181,6 +191,8 @@ extension JobQueueHandler: CustomStringConvertible {
                 label: self.metricsLabel,
                 dimensions: [("name", name), ("status", JobStatus.retried.rawValue)]
             ).increment()
+            // increment queued meter else we will get negative values
+            Meter(label: self.meterLabel, dimensions: [("status", "queued"), ("name", name)]).increment()
```

Comment:
cc @Joannis you were correct all along. `self.queue.push` is does not increment the meter 🥇

Reply:
Ah that's because we call direct to the queue driver to push it back onto the queue

Reply:
Sometimes you use "queued" and sometimes you use `JobStatus.queued.rawValue`. Can we have some consistency

Reply:
> Ah that's because we call direct to the queue driver to push it back onto the queue

It's also possible to have negative values when the server boots. This case is only valid when the job was shutdown while there were still jobs to be processed in the queue. I think queued gauge inc should be moved to one of the push func that'll increment on init, and push to a queue driver. I think the former should only inc if there are jobs in the queue.